### PR TITLE
1455 rest api   add example values in the responses or format responses correctly

### PR DESCRIPTION
**Pull request by spica198**
### Prerequisites

- [ ] Reviewed the [checklist](CHECKLIST.md)

- [ ] Reviewed feedback from the "Sonar Cloud" bot. Note that you have to wait
    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change

Example responses for rest api endpoints added to the the swagger ui

### Verification Process

1. Open Swagger (Run constellation and Tools->Start Rest Server then Display Rest Server Documentation).
2. Check the Example Value of Responses in endpoints.

### Applicable Issues

#1455

### CODE REVIEW

File: CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/GetGraphValues.java [hunk 1]
```diff
@@ -44,6 +44,7 @@ public class GetGraphValues extends RestService {
 
     private static final String NAME = "get_graph_values";
     private static final String GRAPH_ID_PARAMETER_ID = "graph_id";
+    private static final String EXAMPLE_RESPONSES_PATH = "getGrapthValuesExample";
```
Comment (antares1470) [inline CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/GetGraphValues.java]: Graph

File: CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/RenameGraph.java [hunk 2]
```diff
@@ -53,6 +53,7 @@ public class RenameGraph extends RestService {
     private static final String NAME = "rename_graph";
     private static final String NEW_GRAPH_NAME_PARAMETER_ID = "new_graph_name";
     private static final String GRAPH_ID_PARAMETER_ID = "graph_id";
+    private static final String EXAMPLE_RESPONSES_PATH = "renameGrapthExample";
```
Comment (antares1470) [inline CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/RenameGraph.java]: same here

**Review by antares1470 — CHANGES_REQUESTED** [review]

File: CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/GetGraphValues.java (hunk 1 above)
Comment (spica198) [reply CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/GetGraphValues.java]: fixed

File: CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/RenameGraph.java (hunk 2 above)
Comment (spica198) [reply CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/RenameGraph.java]: fixed

**Review by antares1470 — APPROVED** [review]

**Review by Delphinus8821 — APPROVED** [review]

**Comment by Auriga2** [comment]
Response example values look good. But noticed that the `add_recordstore` endpoint no longer works with the example request body, which has changed format. 
image
It works in v3.1 and adds a new node:
image
image

Also is there any way to remove these, although not related to your changes?
image

**Comment by spica198** [comment]
Fixed review comments.. Ready for re-review

**Review by Delphinus8821 — APPROVED** [review]

**Review by Quasar985 — APPROVED** [review]

**Comment by OrionsGuardian** [comment]
Could also include an example output for the `rename_graph` endpoint, ie:
```
{
  "id": "7a9896ac-7fd1-4c8c-b32b-8fb7b847a7fc",
  "previous_name": "analytic graph1",
  "new_name": "renamed graph"
}
```

Also, testing `set_graph` gives an Error response, and testing `reflect_json` gives a null response.
(can possibly be addressed in a separate ticket)

**Comment by spica198** [comment]
> Could also include an example output for the `rename_graph` endpoint, ie:
> 
> ```
> {
>   "id": "7a9896ac-7fd1-4c8c-b32b-8fb7b847a7fc",
>   "previous_name": "analytic graph1",
>   "new_name": "renamed graph"
> }
> ```
> 
> Also, testing `set_graph` gives an Error response, and testing `reflect_json` gives a null response. (can possibly be addressed in a separate ticket)

added response body for rename_graph..
`set_graph` needs a valid graph id in input param or will give an error response.. can test it successfully by giving a valid graph id.
`reflect_json` can reproduce null response on master branch as well.. try to fix it if an easy fix or will raise a separate card.

**Comment by spica198** [comment]
Fixed reflect_json 'null' response
